Editor/BacktraceConfigurationEditor.cs

```diff
@@ -93,11 +95,37 @@ public override void OnInspectorGUI()
                 }
             }
 
-#if UNITY_ANDROID || UNITY_IOS
+            GUIStyle eventAggregationFoldout = new GUIStyle(EditorStyles.foldout);
+            showEventAggregationSettings = EditorGUILayout.Foldout(showEventAggregationSettings, "Crash-free event configuration", eventAggregationFoldout);
+            if (showEventAggregationSettings)
+            {
+                var enableEventAggregation = serializedObject.FindProperty("EnableEventAggregationSupport");
+                EditorGUILayout.PropertyField(
+                    enableEventAggregation,
+                    new GUIContent(BacktraceConfigurationLabels.LABEL_ENABLE_EVENT_AGGREGATION));
+
+                if (enableEventAggregation.boolValue)
+                {
+                    var eventAggregationSubmissionUrl = serializedObject.FindProperty("EventAggregationSubmissionUrl");
+                    EditorGUILayout.PropertyField(
+                        eventAggregationSubmissionUrl,
+                        new GUIContent(BacktraceConfigurationLabels.LABEL_EVENT_AGGREGATION_URL));
+
+                    if (!BacktraceConfiguration.ValidateServerUrl(eventAggregationSubmissionUrl.stringValue))
+                    {
+                        EditorGUILayout.HelpBox("Please insert valid Backtrace server url!", MessageType.Error);
+                    }
+
+                    EditorGUILayout.PropertyField(
+                        serializedObject.FindProperty("TimeIntervalInMs"),
```

Comment:
FWIW this line creates an exception rn in my Unity editor

Reply:
It looks like an issue on my side when I merged two different branches. I will take a look and fix it tomorrow. Sorry for difficulties